variables.tf

```
@@ -202,4 +202,8 @@ variable "cbr_rules" {
   }))
   description = "The list of context-based restrictions rules to create."
```

Cannot be plural here- "rules"

```suggestion
  description = "The context-based restrictions rule to create. Only one rule is allowed."
```

---

Addressed.